### DISCUSSION TITLE
I have already tried all the solutions to the problem and it did not work out, can you please describe in more detail the solution to this problem

### DISCUSSION THREAD

**Discussion by SonGoku4134** [n3d1117/chatgpt-telegram-bot#350] [Q&A]
<img width="654" alt="Снимок экрана 2023-06-19 133215" src="https://github.com/n3d1117/chatgpt-telegram-bot/assets/137000587/19f0dfbc-dae5-4a1d-ade2-7496c6e417de">
if it is possible to give an answer as soon as possible I will be very grateful

**Comment by n3d1117** [comment]
Hey there, as stated in the README you need to do the following:
- In the root folder of the project, rename `.env.example` to `.env`
- Edit `.env` by adding your own telegram bot token (obtained via BotFather) and OpenAI API key (you can find it in OpenAI website)

Good luck!

**Reply by SonGoku4134** [reply]
I have already read this in the discussions, tried it did not help

**Reply by n3d1117** [reply]
Can you share a screenshot of your project folder and contents of .env file?

**Reply by SonGoku4134** [reply]
<img width="514" alt="Снимок экрана 2023-06-19 205913" src="https://github.com/n3d1117/chatgpt-telegram-bot/assets/137000587/92150430-cbc4-4668-8ed6-da13f3d50164">
![Uploading Снимок экрана 2023-06-19 210021.png…]()

**Reply by n3d1117** [reply]
Must be named .env, not .env.env

**Reply by SonGoku4134** [reply]
yes, that's what it's called, it's just that I have the file name extension open
